tests/foreman/cli/test_host.py

```diff
@@ -2216,7 +1912,6 @@ def test_positive_unregister_host_subscription(
     res = rhel_contenthost.register(module_org, None, ak_with_subscription.name, target_sat)
```

Comment:
Do we still need this test? If so, shouldn't the activation key be one without subscriptions?